src/EasyAbp.Abp.EventBus.Boxes.Dtm.Http/EasyAbp/Abp/EventBus/Boxes/Dtm/IDtmMessageBuilder.cs

```diff
@@ -0,0 +1,11 @@
+namespace EasyAbp.Abp.EventBus.Boxes.Dtm;
+
+public interface IDtmMessageBuilder
```

Comment:
Please remove this interface.

If you want to add some extra headers, override the `IDtmMessageManager.CreateDtmMessageInfoModel`.